src/main/java/org/example/service/FileService.java

```diff
@@ -0,0 +1,51 @@
+package org.example.service;
+
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.UrlResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+@Tag(name = "File Service", description = "Service for uploading and downloading files")
+@RestController
+@RequestMapping("/files")
+public class FileService {
```

Comment:
Why this class is Service, but Bank is Controller?
Seems, they should have similar names